Tests/MongoSwiftSyncTests/UnifiedTestRunner/EntityDescription.swift

```diff
@@ -122,7 +122,7 @@ enum EntityDescription: Decodable {
 struct UnifiedTestClient {
     let client: MongoClient
 
-    fileprivate let commandMonitor: UnifiedTestCommandMonitor
+    let commandMonitor: UnifiedTestCommandMonitor
```

Comment:
(changes in this file are because the `assertSame`/`assertDifferentLsidsOnLastTwoCommands` operations needs to be able to access these events)